test/test_tools/test_chunk_serializer.py

```diff
@@ -0,0 +1,67 @@
+import os
+import tempfile
+from pathlib import Path
+
+try:
+    from test.data import TEST_DATA_DIR
+
+    from rdflib import Graph
+    from rdflib.tools.chunk_serializer import serialize_in_chunks
+except Exception:
+    px = str(Path(__file__).absolute().parent.parent.parent)
+    import sys
+
+    sys.path.insert(0, px)
+    from test.data import TEST_DATA_DIR
+
+    from rdflib import Graph
+    from rdflib.tools.chunk_serializer import serialize_in_chunks
```

Comment:
Not that this hurts exactly, but it should not be needed, how do you run test locally? Maybe I could add something to `Taskfile.yml` that could alleviate the problems you experience here.

Reply:
I'm not good with pytest and path issues, so any improvements you can make here... I accept!

Reply:
Changed this now